1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,10 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents ;
+let student = group.map((student) => students.includes(student));
```

Comment:
small thing that won't cause a programming issue but can be visually harder to read; you're using the same variable name `student` to refer to two different things in this line. It works because it's a different scope, but try and avoid [shadowing](https://www.geeksforgeeks.org/variable-shadowing-in-javascript/) where possible